package.json

```diff
@@ -37,7 +37,7 @@
   "devDependencies": {
     "babel": "^5.6.14",
     "babel-jest": "^5.3.0",
-    "jest-cli": "^0.4.0",
+    "jest-cli": "^0.5.0",
```

Comment:
I don't quite understand how npm's versioning works but jest only started working with node 4 at around 0.5.6 or something. It might make sense to specify `0.5.10` as the minimum requirement.